rnd/nextgenautogpt/__main__.py

```diff
@@ -0,0 +1,39 @@
+import multiprocessing as mp
+from typing import Any
+
+import nextgenautogpt.manager.manager as mod_manager
+import nextgenautogpt.server.server as mod_server
+
+
+def main() -> None:
+    # Create queues/pipes for communication
+    server_to_manager: mp.Queue[Any] = mp.Queue()
```

Comment:
mp.queue will not allow any other python process that are not forked from the main to use it, shall we consider using ipcqueue? 
https://pypi.org/project/ipcqueue/

Reply:
That project has not been updated for 5 ish years and only supports up to python 3.6

Reply:
Even better: https://zeromq.org/

This is like sqlite for message queue.